samples/csharp/roslyn-sdk/SyntaxQuickStart/HelloSyntaxTree/Program.cs

```diff
@@ -32,41 +32,41 @@ static void Main(string[] args)
         {
             // <Snippet2>
             SyntaxTree tree = CSharpSyntaxTree.ParseText(programText);
-            var root = tree.GetRoot() as CompilationUnitSyntax;
+            CompilationUnitSyntax root = tree.GetCompilationUnitRoot();
             // </Snippet2>
 
             // <Snippet3>
-            WriteLine($"The tree is a {root.Kind().ToString()} node.");
+            WriteLine($"The tree is a {root.Kind()} node.");
             WriteLine($"The tree has {root.Members.Count} elements in it.");
             WriteLine($"The tree has {root.Usings.Count} using statements. They are:");
             foreach (UsingDirectiveSyntax element in root.Usings)
-                WriteLine($"\t{element.Name.ToString()}");
+                WriteLine($"\t{element.Name}");
             // </Snippet3>
 
             // <Snippet4>
             MemberDeclarationSyntax firstMember = root.Members[0];
-            WriteLine($"The first member is a {firstMember.Kind()}");
-            NamespaceDeclarationSyntax helloWorldDeclaration = firstMember as NamespaceDeclarationSyntax;
+            WriteLine($"The first member is a {firstMember.Kind()}.");
+            var helloWorldDeclaration = (NamespaceDeclarationSyntax)firstMember;
             // </Snippet4>
 
             // <Snippet5>
-            WriteLine($"There is {helloWorldDeclaration.Members.Count} member declared in this namespace.");
-            WriteLine($"The first member is a {helloWorldDeclaration.Members[0].Kind()}");
+            WriteLine($"There are {helloWorldDeclaration.Members.Count} members declared in this namespace.");
```

Comment:
I went against a review comment (https://github.com/dotnet/docs/pull/4289#discussion_r166084483) here. I think incorrect grammar is better than "double-think" code (if it knows `Count` is 1, it could just use `There is 1 member`; if it doesn't know it's 1, it shouldn't use singular).

Reply:
If we were really smart, we'd do something like:
```csharp
var count = hellowWorldDeclarationMembers.Count;
WriteLine($"There {(count == 1 ? "is" : "are")} {count} member{(count != 1 ? "s" : "")} declared in this namespace.");
```
But, I'm afraid that sacrifices the learning aspect in favor of correctness. Thoughts?

Reply:
@BillWagner Yeah, that's what I thought: writing more complicated code is not worth it here.